### PR TITLE
Putting some distance between Jenkins & CI only

### PR DESCRIPTION
**Pull request by batmat**
Next steps will be modifying the URL when jenkins.io is ready.
#2030 and @oleg-nenashev's comment made me think about that PR.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:+1: 
Would be great to rename Twitter account as well. CC @rtyler

**Comment by rtyler** [comment]
We're not renaming account handles anywhere, they're simply not available unfortunately. There is some more text I can update on twitter, though the "Name" had been changed ages ago

**Comment by batmat** [comment]
We should at least try and choose a new hashtag (another than #jenkinsci at least) and advertise it.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 [![][ButlerImage]][website] 
 
 # About
-In a nutshell, Jenkins CI is the leading open-source continuous integration server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
+In a nutshell, Jenkins is the leading open-source automation server. Built with Java, it provides over 1000 plugins to support building and testing virtually any project.
```
Comment (jtnord) [inline README.md]: Sounds like a ci server.
Perhaps "... building, testing, and deploying..." to make it more than just ci?

Or use the text from https://wiki.jenkins-ci.org/display/JENKINS/Meet+Jenkins

Comment (batmat) [inline README.md]: Done. After reading the wiki, I felt this was still to software delivery related when we were agreeing here on an even more generic automation application.

**Comment by batmat** [comment]
Just amended another paragraph after @jtnord's comment. Let me know what you think :)

**Comment by jtnord** [comment]
:+1:

**Comment by amuniz** [comment]
LGTM :+1:

**Pull request merged**